src/app/style/base/_input.scss

```diff
@@ -36,6 +37,11 @@ select {
             margin-top: 1.4rem;
         }
     }
+
+    &:-webkit-autofill {
+        box-shadow: 0 0 0 30px var(--input-background-color) inset !important;
+        background-color: var(--input-background-color) !important;
+    }
```

Comment:
The project uses `autoprefixer`, I believe you have no need to prefix for specific browsers separately.